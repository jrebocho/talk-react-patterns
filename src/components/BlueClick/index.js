import withBlueClick from '../BlueClickHOC'
import Stateless from '../Stateless'

export default withBlueClick(Stateless)
