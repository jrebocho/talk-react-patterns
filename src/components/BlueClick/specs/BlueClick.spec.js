import React from 'react'
import { shallow } from 'enzyme'
import BlueClick from '../../BlueClick'

describe('<BlueClick />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<BlueClick />)
    expect(wrapper).toMatchSnapshot()
  })
})
