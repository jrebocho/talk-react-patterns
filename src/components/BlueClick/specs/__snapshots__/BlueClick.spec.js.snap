// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlueClick /> renders correctly 1`] = `
<div
  className="blue-box"
  onClick={[Function]}
>
  <Stateless
    on={false}
  />
</div>
`;
